source/code/plugins/changetracking_lib.rb

```diff
@@ -12,6 +12,8 @@ def self.log= (value)
         @@log = value
     end
 
+    @@force_send_last_upload = Time.now
```

Comment:
suggest to use utc time
shall we persist the last upload time? otherwise, every restart will reset the time. The DSC configuration triggers a restart, whenever there is a change.